server/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy

```diff
@@ -111,3 +111,7 @@ grant codeBase "${codebase.netty-transport}" {
    permission java.util.PropertyPermission "sun.nio.ch.bugLevel", "write";
    permission java.net.SocketPermission "*", "accept,connect";
 };
+
+grant {
+  permission java.net.SocketPermission "127.0.0.1:*", "connect, resolve";
```

Comment:
Hrm, why do we need this?

Reply:
The integration and unit tests attempt to connect to the socket the readiness service makes. So I needed to give the tests permission to connect to the socket.